src/ensemble/ensemble_problems.jl

```diff
@@ -16,7 +16,11 @@ DEFAULT_REDUCTION(u, data, I) = append!(u, data), false
 DEFAULT_VECTOR_PROB_FUNC(prob, i, repeat) = prob[i]
 function EnsembleProblem(prob::AbstractVector{<:AbstractSciMLProblem}; kwargs...)
     # TODO: @invoke
-    invoke(EnsembleProblem, Tuple{Any}, prob; prob_func=DEFAULT_VECTOR_PROB_FUNC, kwargs...)
+    invoke(EnsembleProblem,
+        Tuple{Any},
+        prob;
+        prob_func = DEFAULT_VECTOR_PROB_FUNC,
+        kwargs...)
```

Comment:
What's up with this formatting? This doesn't look like the right formatter.

Reply:
@isaacsas do you know what could be causing this?

Reply:
The formatting? No idea. There weren’t any recent changes, and the last change was for function declaration signatures only right?

Reply:
Maybe this was a manual reformatting that rejected the auto format?

Reply:
@ErikQQY can you please minimize the change here and do the formatting in a different PR?

Reply:
done